tools/clang/test/DXC/embed_dbg.test

```diff
@@ -2,7 +2,8 @@
 // RUN: %dxc /T ps_6_0 %S/Inputs/smoke.hlsl /Zi /Qembed_debug /Fo %t.embedpdb
 // RUN: %dxc -dumpbin %t.embedpdb | FileCheck  %s --check-prefix=EMBED_PDB
 
-// RUN: ls -1 %T | not grep "\\.pdb$"
+// RUN: ls -1 | FileCheck %s --check-prefix=PDB_NAME
+// PDB_NAME-NOT:.pdb{{$}}
```

Comment:
Is the point of this to get rid of the `grep` call? How are we going to handle tests from upstream that rely on `grep`?

Reply:
I think it bears adding a comment explaining that